src/audio/selector/selector.c

```diff
@@ -882,10 +882,10 @@ static int selector_prepare(struct processing_module *mod)
 	 * proper number of channels [1] for selector to actually
 	 * reduce channel count between source and sink
 	 */
-	comp_info(dev, "selector_prepare(): sourceb->schannels = %u",
-		  source_c->stream.channels);
-	comp_info(dev, "selector_prepare(): sinkb->channels = %u",
-		  sink_c->stream.channels);
+	comp_dbg(dev, "selector_prepare(): sourceb->schannels = %u",
+		 source_c->stream.channels);
+	comp_dbg(dev, "selector_prepare(): sinkb->channels = %u",
+		 sink_c->stream.channels);
```

Comment:
I haven't worked with selector yet, but I would imagine that this could be useful too, you'd save some if you merge the two into one